tensorflow/c/c_api_test.cc

```diff
@@ -1543,6 +1543,33 @@ TEST(CAPI, TestBitcastFrom_Reshape) {
   TF_DeleteTensor(b);
 }
 
+TEST(CAPI, TestFromProto) {
+  Tensor t_cc(DT_FLOAT, TensorShape({2, 3}));
+  t_cc.flat<float>().setConstant(1.0);
+  tensorflow::TensorProto t_proto;
+  t_cc.AsProtoField(&t_proto);
+
+  TF_Buffer* t_buffer = TF_NewBuffer();
+  tensorflow::MessageToBuffer(t_proto, t_buffer);
+
+  const int num_bytes = 6 * sizeof(float);
+  float* values =
+      reinterpret_cast<float*>(tensorflow::cpu_allocator()->AllocateRaw(
+          EIGEN_MAX_ALIGN_BYTES, num_bytes));
+  int64_t dims[] = {2, 3};
+  bool deallocator_called = false;
+  TF_Tensor* t_c = TF_NewTensor(TF_FLOAT, dims, 2, values, num_bytes,
+                                &Deallocator, &deallocator_called);
+
+  TF_Status* status = TF_NewStatus();
```

Comment:
This is leaking. Please add `TF_DeleteStatus(status);`

Reply:
Done